src/test/unit/regions/RegionTestUtilities.cpp

```diff
@@ -0,0 +1,529 @@
+/* ---------------------------------------------------------------------
+ * Numenta Platform for Intelligent Computing (NuPIC)
+ * Copyright (C) 2018, Numenta, Inc.  Unless you have an agreement
+ * with Numenta, Inc., for a separate license for this software code, the
+ * following terms and conditions apply:
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses.
+ *
+ * http://numenta.org/licenses/
+ 
+
+
+ * Author: David Keeney, July, 2018
+ * ---------------------------------------------------------------------
+ */
+
+
+#define VERBOSE                                                                \
+  if (verbose)                                                                 \
+  std::cerr << "[          ] "
```

Comment:
Can this cause error that the define is not on a single line?

Reply:
> Can this cause error that the define is not on a single line?

The '\' on the far right makes it one line.
But I can make it a single line.  I think I did other places.